src/disk/state.rs

```diff
@@ -243,10 +243,43 @@ pub struct Xmrig {
     pub token: String,
 }
 
-#[derive(Clone, Eq, PartialEq, Debug, Deserialize, Serialize, Default)]
+#[derive(Clone, PartialEq, Debug, Deserialize, Serialize, Default)]
```

Comment:
Rust default of a bool is false but simple should be true the first time the value is created in the state. Else the XvB tab will open to advanced tab at next update. Simple tab should always be preferred as default.